bindings/rust/aws-lc-fips-sys-template/README.md

```diff
@@ -0,0 +1,44 @@
+# aws-lc-fips-sys
+
+[![crates.io](https://img.shields.io/crates/v/aws-lc-fips-sys.svg)](https://crates.io/crates/aws-lc-fips-sys)
+
+Low level AWS-LC FIPS bindings for the Rust programming language. We do not recommend directly relying on these bindings.
+
+[Documentation](https://github.com/awslabs/aws-lc).
+
+## Release Support
+
+This crate pulls in the source code of the latest AWS-LC FIPS branch to build with it. Bindings for platforms we officially support are pre-generated.
+The platforms which `aws-lc-fips-sys` builds on is limited to the platforms where the AWS-LC FIPS static build is supported.
+
+### Pregenerated Bindings Availability
+
+CPU|OS
+-------------|-------------
+x86-64|Ubuntu
+arm-64|Ubuntu
```

Comment:
You mention "Ubuntu" specifically, but the pregenerated bindings apply to users on any Linux distribution.
```
  CPU | OS
------|-----
x86-64|Linux
arm-64|Linux
```